doc/source/examples/dem/rectangular-hopper/rectangular-hopper.rst

```diff
@@ -0,0 +1,210 @@
+==================================
+Rectangular hopper
+==================================
+
+This example simulates the filling and discharge of particles in a rectangular hopper.
+We set up this simulation based on the simulation of Anand et al. `[1] <https://doi.org/10.1016/j.ces.2008.08.015>`_. It is recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_ for more detailed information on the concepts and physical meanings of the parameters in Lethe-DEM.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Floating walls
+- Gmsh grids
+- Python post-processing script using PyVista
+
+
+Files used in this example
+----------------------------
+``/examples/dem/3d-rectangular-hopper/hopper.prm``
+
+
+Description of the case
+-----------------------
+
+This simulation consists of two stages: filling (0-4 s) and discharge (4-7.5 s) of particles. Anand et al. uses periodic boundaries in the z axis allowing to use a thin width for simulation. This case do not use periodic boundary in Lethe. Prior minimizing the impact of collision of particle with walls in z axis, the width and the number of particle were multiplied 6 times. This corresponds to a width of 15 times the particle diameter.
```

Comment:
```suggestion
This simulation consists of two stages: filling (0-4 s) and discharge (4-7.5 s) of particles. Anand et al. uses periodic boundaries in the z axis allowing to use a thin width for simulation.  Since those are not available at the time in Lethe, we do not consider periodic boundaries. To minimize the impact of collision of particle with walls along the z axis, the width and the number of particle were multiplied by 6 (YYY particles instead of ZZZ). This corresponds to a width of 15 times the particle diameter.
```

Correct number of particles used